### ISSUE TITLE
On Windows, RemoteWebDriver.quit() is always destroying forcefully?

### ISSUE THREAD

**Issue by benlei-neustar** [SeleniumHQ/selenium#614]
I'm constantly receiving the below log output:
INFO 06/04 22:43:04 o.o.s.o.UnixProcess  - Command failed to close cleanly. Destroying forcefully (v2). org.openqa.selenium.os.UnixProcess$SeleniumWatchDog@6527b9ac

This happens whenever I do a `driver.quit()` (which is a `RemoteWebDriver`). This problem is not showing with Selenium 2.45.0.

This will stop the output from happening, but I am not sure if this is the right way to go:
```java
diff --git a/java/client/src/org/openqa/selenium/os/UnixProcess.java b/java/client/src/org/openqa/selenium/os/UnixProcess.java
index cd7af38..47d6575 100644
--- a/java/client/src/org/openqa/selenium/os/UnixProcess.java
+++ b/java/client/src/org/openqa/selenium/os/UnixProcess.java
@@ -119,9 +119,12 @@ public int destroy() {
       if (!isRunning()) {
         return getExitCode();
       }
+
+      watchdog.destroyHarder();
+    } else {
+      watchdog.destroy(); // don't want the error message
     }
 
-    watchdog.destroyHarder();
     watchdog.waitForTerminationAfterDestroy(1, SECONDS);
     if (!isRunning()) {
       return getExitCode();
@@ -244,6 +247,10 @@ private void waitForTerminationAfterDestroy(int duration, TimeUnit unit) {
 
     private void destroyHarder() {
       log.info("Command failed to close cleanly. Destroying forcefully (v2). " + this);
+      destroy();
+    }
+
+    private void destroy() {
       Process ourProc = process;
       ProcessUtils.killProcess(ourProc);
     }
```

the post that introduced the plugin container fix: https://code.google.com/p/selenium/issues/detail?id=7506

**Comment by pankajsukale38** [comment]
Facing the same. Can any one respond please.

**Comment by barancev** [comment]
It's just an INFO message. Well, may be it is worth to hide it on the DEBUG level :)

**Comment by benlei-neustar** [comment]
It is indeed just an INFO message, but the message can mislead people that something wrong has occurred when nothing wrong did occur (unless it was a non-windows process).

**Comment by 1tft** [comment]
Facing the same problem. Using Firefox ESR 38 and Firefox ESR 31.
With Selenium 2.44.0 no such message appeared.

**Comment by barancev** [comment]
With Selenium 2.44 Firefox crashed on quit in plugin-container module.
I think that an info message is not a worse alternative :)

**Comment by timja** [comment]
Is it expected to always have to destroy harder on Windows?

**Comment by barancev** [comment]
Fixed by commit 7a5e890ac2d37c76ac972ee22cfbc8aa89f83742
